fuzzing/private/fuzz_test.bzl

```diff
@@ -133,6 +133,10 @@ def fuzzing_decoration(
         tags = ["manual"],
     )
 
+# Use += rather than append to allow users to pass in select() expressions for
+# deps, which only support concatenation with +.
+# Workaround for https://github.com/bazelbuild/bazel/issues/14157.
+# buildifier: disable=list-append
```

Comment:
This doesn't look it belongs here, as there is no "+=" being used on the next line. Move close to where it's relevant?

Reply:
Disabling this lint check seems overly broad at the function level.

Reply:
I changed this and squashed the commits.